utils/html.go

```diff
@@ -116,5 +116,32 @@ func (t *HTMLTemplate) RenderToWriter(w http.ResponseWriter) error {
 		l4g.Error(T("api.api.render.error"), t.TemplateName, err)
 		return err
 	}
+
 	return nil
 }
+
+func TranslateAsHtml(t i18n.TranslateFunc, translationID string, args ...interface{}) template.HTML {
+	safeArgs := make([]interface{}, len(args))
+
+	for i, arg := range args {
+		safeArgs[i] = escapeForHtml(arg)
+	}
+
+	return template.HTML(t(translationID, safeArgs...))
+}
+
+func escapeForHtml(arg interface{}) interface{} {
+	switch typedArg := arg.(type) {
+	case string:
+		return template.HTMLEscapeString(typedArg)
+	case map[string]interface{}:
+		safeArg := make(map[string]interface{}, len(typedArg))
+		for key, value := range typedArg {
+			safeArg[key] = escapeForHtml(value)
+		}
+		return safeArg
+	default:
+		l4g.Warn("Unable to escape value for HTML template %v", arg)
+		return arg
```

Comment:
If @coreyhulen's suggestion is possible, we should do that, but otherwise I would be slightly in favor of returning something safe here like "" or "???". That would be the "secure by default" way.